components/Dropdown/DropdownItem.module.scss

```diff
@@ -1,10 +1,16 @@
 .item {
+    width: max-content;
+    min-width: 100%;
+    max-width: 100%;
```

Comment:
That one looks... interesting... 🤔 
Do you have an article or something that describes how this one works? I'd be curious to know :)